tools/publish_release.sh

```diff
@@ -108,17 +108,12 @@ create_javadoc() {
   cd $HERE
 }
 
-create_native_debug_symbols_package() {
```

Comment:
Don't we upload these anymore?

Reply:
We do, but it is handled automatically as part of the `upload_debug_symbols` task